packages/core/strapi/src/commands/index.ts

```diff
@@ -92,6 +99,10 @@ const buildStrapiCommand = async (argv: string[], command = new Command()) => {
   command.helpOption('-h, --help', 'Display help for command');
   command.addHelpCommand('help [command]', 'Display help for command');
 
+  // Debug and silent options
+  command.option('-d, --debug', 'Output extra debugging');
```

Comment:
Same comment as above adding this globally when it won't be supported globally will create invalid help